src/app/components/media/CreateRelatedMediaDialog.js

```diff
@@ -106,7 +106,7 @@ class CreateRelatedMediaDialog extends React.Component {
         </DialogContent>
         <DialogActions>
           <Button id="create-media-dialog__dismiss-button" onClick={this.props.onDismiss}>
-            {this.props.intl.formatMessage(globalStrings.cancel)}
+           <FormattedMessage{...globalStrings.cancel} />
```

Comment:
Please add a space between `FormattedMessage` and `{`. Also please remove the `injectIntl` from the `import` statement and its respective usage.